chart/compass/charts/connector/templates/service-role-binding.yaml

```diff
@@ -11,27 +11,22 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
 spec:
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+      release: {{ .Release.Name }}
+  action: ALLOW
   rules:
-    - services: ["{{ template "fullname" . }}-internal.{{ .Release.Namespace }}.svc.cluster.local"]
-      paths: ["*"]
-      methods: ["*"]
----
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
-metadata:
-  name: {{ template "fullname" . }}-internal
-  namespace: {{ .Release.Namespace }}
-  labels:
-    release: {{ .Release.Name }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    app.kubernetes.io/name: {{ template "name" . }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-spec:
-  subjects:
-{{ tpl (toYaml .Values.istio.rbac.connectorInternalApi.subjects | indent 4) . }}
-  roleRef:
-    kind: ServiceRole
-    name: {{ template "fullname" . }}-internal
----
+    - from:
+      - source:
+          principals:
+          - {{ tpl .Values.istio.rbac.connectorInternalApi.compassDirector . }}
+          - {{ tpl .Values.istio.rbac.connectorInternalApi.compassConnectorTests . }}
+    - to:
+      - operation:
+          ports:
```

Comment:
as we are not using services for rules now, do we need to keep both clusterip services (connector internal service and connector external service?) or can one be removed ?

Reply:
as described in https://github.com/kyma-incubator/compass/pull/1536/files#r495713387, the only ports that should stay here are ones from the external service.

Reply:
thanks, my question was actually do we need the clusterIP connector-internal-service ? Can we just put all four ports on connector-service clusterip?

Reply:
Oh, you are right. If nothing else than Istio depends on it, then we can put all ports in one service. But I guess that some applications use the service names as addresses (for example: `compass-connector-internal.compass-system.svc.cluster.local` for connecting to internal endpoints), so we would need to know where this should be changed.

Also, it may be only my opinion, but I think that the separation for internal/external ports with services is a good and "foolproof" idea. The probability of making a mistake and for example exposing internal ports by accident is IMO lower when you see that the ports are used by a service called `internal`.

Reply:
Ok, I see that you already did a merge of the two services. Please make sure that `chart/compass/charts/connector/templates/tests/test-connector.yaml` and `chart/compass/charts/director/templates/deployment.yaml` were the only places where `-internal` prefix should be removed.

Reply:
i didnt see any other places. My problem with the two services is that after the migration to the new authz rule there is no explicit "binding" that the ports defined a service called internal are actually internal (and there was such a binding with the previous istio cr). Thats why i thought it would make sense